doc/usecases/snippet_sharing/UC_edit_snippet.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Edit configuration snippet
+
+## Summary
+
+Title: Edit configuration snippet
+Scope: Entry Management
+Level: User Goal
+Actors: Authenticated user (snippet owner), Moderator
+Brief: Users can edit their own database entries. Moderators can edit all configuration snippets in the database, also the ones of other users.

```

Comment:
To do so, I would need to retrieve a mapping between plugins and formats from the API, but I think that's easily possible, so why not.